Automattic/WP-Job-Manager
# "Load previous listings" bug fix #1604

Fixes #1604 

#### Changes proposed in this Pull Request:

As stated in the issue description the text "Load previous listings" incorrectly (unnecessarily) appears on the page when we click on the "Load More Listings" then refresh the Job Listings Page.

This is due to a few lines which are not necessary in the code, although I would suggest to open another ticket, which would be an enhancement to create a "Load Less Listings" link. Removing these lines should not affect any functionality of the page, except resulting in the expected page output.

With the current suggested changes the page loads as expected. (tested with 0 jobs, and jobs>jobs list per page). 
The load spinner shows, jobs are displayed correctly and stay displayed the same way on refresh.

Solution suggestion: Remove lines 30-44

Explanation of line removals (also added as code comments):

- Line 31: It's not necessary to add the 'load' class because the onClick handler adds this class (line 207 in the ajax-filter.js file)
- Line 35-40: These lines are responsible for outputting and handling the text that was requested to be removed from the page (also the set of matched elements is not well defined in the .wrap() jQuery element, so it is not functional)
- Line 42: This line modifies the output data unnecessarily.

#### Testing instructions:

As in the issue description:

1. Make sure you have more jobs that you are showing on the first page of job listings, so that a "Load More Listings" button or link appears
2. Click it to load more listings
3. Refresh the page

Page is expected to keep displaying the expanded job list without any incorrect text appearing.
